scripts/minikube.sh

```diff
@@ -53,6 +53,33 @@ function install_kubectl() {
     curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/"${KUBE_VERSION}"/bin/linux/amd64/kubectl && chmod +x kubectl && mv kubectl /usr/local/bin/
 }
 
+function deploy_rook() {
+    ROOK_URL=https://raw.githubusercontent.com/rook/rook/$ROOK_VERSION/cluster/examples/kubernetes/ceph
+
+    kubectl create -f $ROOK_URL/common.yaml
+    kubectl create -f $ROOK_URL/operator.yaml
+    kubectl create -f $ROOK_URL/cluster-test.yaml
+    kubectl create -f $ROOK_URL/toolbox.yaml
+
+    for ((retry=0; retry<=${ROOK_DEPLOY_TIMEOUT}; retry++)); do
+        echo "Wait for rook deploy... ${retry}s"
+        sleep 1
```

Comment:
move this to a new file, so people who want to deploy rook for testing can use it  easily

Reply:
If we move to a new file (maybe scripts/deploy_rook.sh), testing will be like this.

```shell
./scripts/minikube.sh up
./scripts/depoy_rook.sh
(...)
```

how about just don't move and use minikube.sh?
```shell
./scripts/minikube.sh up
./scripts/minikube.sh deploy-rook
(...)
```

Reply:
let it be, it will be more useful for the developers who will bring up there own Kubernetes cluster other than minikube

Reply:
> let it be, it will be more useful for the developers who will bring up there own Kubernetes cluster other than minikube

hmm, that make sense. I will fix it.

Reply:
I moved the code like your suggestion. but I keep ./scripts/minikube.sh deploy-rook. now, user who want to deploy own cluster can run ./scripts/rook.sh file but others can run just ./scripts/minikube.sh.